skimage/registration/_optical_flow.py

```diff
@@ -0,0 +1,215 @@
+# coding: utf-8
+"""TV-L1 optical flow algorithm implementation.
+
+"""
+
+from functools import partial
+import numpy as np
+from scipy import ndimage as ndi
+from skimage.transform import warp
+
+from .utils import coarse_to_fine
```

Comment:
These should be in a private module `_utils` to make it clear that they should not be used publicly. I also might call the module `_optical_flow_utils` to make it clearer what the file is when browsing the repo.